cairo-libs/packages/cometbft/src/ics23.cairo

```diff
@@ -1,219 +0,0 @@
-use protobuf::types::message::{
```

Comment:
Moved into the `packages/ics23` library, and the enums have been revised to include only the operations (variants) that are already supported on Starknet or those we have an implementation for.